StandaloneMmPkg/Drivers/StandaloneMmIplPei/StandaloneMmIplPei.c

```diff
@@ -147,12 +147,41 @@ LocateMmCoreFv (
   OUT VOID                  **MmCoreImageAddress
   )
 {
-  EFI_STATUS           Status;
-  UINTN                FvIndex;
-  EFI_PEI_FV_HANDLE    VolumeHandle;
-  EFI_PEI_FILE_HANDLE  FileHandle;
-  EFI_PE32_SECTION     *SectionData;
-  EFI_FV_INFO          VolumeInfo;
+  EFI_STATUS               Status;
+  UINTN                    FvIndex;
+  EFI_PEI_FV_HANDLE        VolumeHandle;
+  EFI_PEI_FILE_HANDLE      FileHandle;
+  EFI_PE32_SECTION         *SectionData;
+  EFI_FV_INFO              VolumeInfo;
+  MM_CORE_FV_LOCATION_PPI  *MmCoreFvLocation;
+
+  Status = PeiServicesLocatePpi (&gMmCoreFvLocationPpiGuid, 0, NULL, (VOID **)&MmCoreFvLocation);

```

Comment:
Add comments to explain that the code below is to use FV indicated by MmCoreFvLocation PPI directly.
And clarify that it's the producer's responsibility that MmCoreFvLocation PPI reports the correct FV that contains MmCore.

Reply:
ok